agent/src/iconfucius/cli/balance.py

```
@@ -844,62 +886,75 @@ def run_all_balances(bot_names: list, token_id: str = "29m8",
+    else:
+        holdings_display = _format_holdings_table(
+            all_data, btc_usd_rate, wallet_balance, wallet_pending,
+            wallet_withdrawal,
+        )
+        display_lines = wallet_lines + [
+            "",
+            "=" * 60,
+            "Bot Holdings at Odin.Fun",
+            "=" * 60,
+            holdings_display,
+        ]
```

Ruff (0.15.2)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)